src/library_webgpu.js

```diff
@@ -1512,6 +1512,13 @@ var LibraryWebGPU = {
     var color = WebGPU.makeColor(colorPtr);
     pass["setBlendColor"](color);
   },
+  wgpuRenderPassEncoderSetIndexBuffer: function(passId, bufferId, format, {{{ defineI64Param('offset') }}}, size) {
```

Comment:
There might be a way to define this so it literally points to the same code, but not going to bother figuring it out because we'll remove the other copy soon.